packages/baml/src/services/baml-service.ts

```
@@ -45,7 +48,7 @@ export async function runOpenAIChat(message: string): Promise<string> {
     }),
   });
 
-  const data = await res.json();
+  const data = await res.json() as any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)